demisto_sdk/commands/integration_diff/integration_diff_detector.py

```diff
@@ -0,0 +1,155 @@
+import os
+
+from demisto_sdk.commands.common.tools import (get_yaml, print_error,
+                                               print_success)
+
+
+class IntegrationDiffDetector:
+
+    def __init__(self, new: str = '', old: str = ''):
+
+        if not os.path.exists(new) or not os.path.exists(old):
+            print_error('No such file or directory.')
+
+        self.new = new
+        self.old = old
+
+        self.fount_missing = False
+        self.missing_details_report: dict = {
+            'commands': [],
+            'arguments': [],
+            'outputs': []
+        }
+
+    def check_diff(self):
```

Comment:
add type annotations and docstring - this applies to all functions in this file

Reply:
done